src/ride_type.cpp

```diff
@@ -226,6 +226,23 @@ const RideType *RideInstance::GetRideType() const
  * Immediately remove this ride from all voxels it currently occupies.
  */
 
+/**
+ * \fn void RideInstance::InsertIntoWorld()
+ * Ensure that this shop is linked into the voxels it is meant to occupy.
+ */
+
+/**
+ * \fn void ShouldDrawPiece::InsertIntoWorld()
+ * Whether the specified piece of the ride should be visible to the player.
+ * @param voxel_number The instance data of the piece to check.
+ * @return `false` if the ride piece is meant to be fully invisible; `true` otherwise.
+ */
+bool RideInstance::ShouldDrawPiece(uint16 voxel_number) const
+{
+	// By default, all pieces are drawn, unless a subclass implements a special case
```

Comment:
Comments are normal sentences, they start with an upper case letter (which you do), and end with a dot.